native_client/java/setup.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/sh
```

Comment:
we should not require this kind of script

Reply:
A makefile would be a better solution probably. But I don't know how to work with a Makefile, and have not figured out yet how to make CMake interact with Gradle without using Android